src/MoBi.UI/Views/SelectReferenceView.cs

```diff
@@ -64,6 +64,12 @@ public bool EnableMultiSelect
          get => _treeView.OptionsSelection.MultiSelect;
       }
 
+      public void SetLayoutItemLocalisationVisible(bool visible) 
+         => layoutItemLocalisation.Visibility = visible ? LayoutVisibility.Always : LayoutVisibility.Never;
+
+      public void SetRelativePathRadioButtonEnabled(bool enabled)
+         => radioGroupReferenceType.Properties.Items.GetItemByValue(ObjectPathType.Relative).Enabled = enabled;
```

Comment:
There's a static class from DevExpress `LayoutVisibilityConvertor` that handles the boolean conversions like this.